### PR TITLE
add erroneous file

### PR DESCRIPTION
**Pull request by mjenrungrot**

### CODE REVIEW

File: problems.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# TODO: Fix this whole function
+def printHi3TimesNTimes(n)
```
Comment (nkadonaga) [inline problems.rb]: Consider using a better name

File: problems.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  for i in 0..n
+    for j in 0..3
+      puts "Hi"
```
Comment (nkadonaga) [inline problems.rb]: Consider storing "Hi" in a variable to avoid magic values

File: problems.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# TODO: Fix this whole function
+def printHi3TimesNTimes(n)
+  for i in 0..n
```
Comment (nkadonaga) [inline problems.rb]: Consider combining the two nested loops into one loop for code clarity

File: problems.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# TODO: Fix this whole function
```
Comment (nkadonaga) [inline problems.rb]: Remember to remove TODO comment

**Review by nkadonaga — CHANGES_REQUESTED** [review]
💃

File: problems.rb (hunk 2 above)
Comment (mjenrungrot) [reply problems.rb]: Done!

File: problems.rb (hunk 3 above)
Comment (mjenrungrot) [reply problems.rb]: Done!

**Review by nkadonaga — APPROVED** [review]
LGTM 👍

**Comment by mjenrungrot** [comment]
👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍

**Pull request merged**